src/components/Page.vue

```diff
@@ -279,6 +282,10 @@ export default {
 			await this.setEmoji(this.currentPage.parentId, this.currentPage.id, emoji)
 		},
 
+		async unselectPageEmoji() {
+			await this.setPageEmoji('')
```

Comment:
I don't think there's a reason to use async await here. How about this:
```suggestion
		unselectPageEmoji() {
			return this.setPageEmoji('')
```

If `this.setPageEmoji` is async i.e. returns a promise so will `unselectPageEmoji`.

Reply:
Thanks, sounds good to me :ok_hand: